symfony/plugins/orangehrmPerformancePlugin/test/fixtures/testcases/PerformanceTrackerAPITest.yaml

```diff
@@ -0,0 +1,230 @@
+          empNumber: 1
+          lastName: 'Abbey'
+          firstName: 'Kayla'
+    meta:
+      total: 1
+  'Get all trackers and sort by addedDate':
+    userId: 1
+    services:
+      performance.performance_tracker_service: \OrangeHRM\Performance\Service\PerformanceTrackerService
+    attributes: [ ]
+    query:
+      sortField: 'performanceTracker.addedDate'
+    data:
+      0:
+        id: 3
+        trackerName: 'test tracker3 name'
+        addedDate: '2011-12-10'
+        modifiedDate: null
+        status: 1
+        employee:
+          empNumber: 4
+          lastName: 'Sherlock'
+          firstName: 'Sherin'
+      1:
+        id: 1
+        trackerName: 'test tracker name'
+        addedDate: '2011-12-12'
+        modifiedDate: null
+        status: 1
+        employee:
+          empNumber: 1
+          lastName: 'Abbey'
+          firstName: 'Kayla'
+      2:
+        id: 2
+        trackerName: 'test tracker2 name'
+        addedDate: '2011-12-12'
+        modifiedDate: null
+        status: 1
+        employee:
+          empNumber: 2
+          lastName: 'Abel'
+          firstName: 'Ashley'
+    meta:
+      total: 3
+
+Create:
+  'Create performance tracker':
+    userId: 1
+    services:
+      performance.performance_tracker_service: \OrangeHRM\Performance\Service\PerformanceTrackerService
+    now:
+      datetime: '2022-02-03'
+    attributes: [ ]
+    body:
+      trackerName: 'New tracker 345'
+      empNumber: 3
+      reviewers: [ 2,4 ]
+    query: [ ]
+    data:
+      id: 4
+      trackerName: 'New tracker 345'
+      addedDate: '2022-02-03'
+      modifiedDate: null
+      status: null
+      employee:
+        empNumber: 3
+        lastName: 'Saputhanthri'
+        firstName: 'Renukshan'
+    meta: [ ]
+
+Update:
+  'update all details':
+    userId: 1
+    services:
+      performance.performance_tracker_service: \OrangeHRM\Performance\Service\PerformanceTrackerService
+      core.normalizer_service: \OrangeHRM\Core\Service\NormalizerService
+    now:
+      datetime: '2022-02-03'
+    attributes:
+      id: 1
+    body:
+      trackerName: 'Tracker updated second time'
+      empNumber: 3
+      reviewers: [ 1,2 ]
+    query: [ ]
+    data:
+      id: 1
+      trackerName: 'Tracker updated second time'
+      addedDate: '2011-12-12'
+      modifiedDate: '2022-02-03'
+      empNumber: 3
+      lastName: 'Saputhanthri'
+      firstName: 'Renukshan'
+      terminationId: null
+      reviewers:
+        0:
+          id: 5
+          empNumber: 1
+          lastName: 'Abbey'
+          firstName: 'Kayla'
+        1:
+          id: 6
+          empNumber: 2
+          lastName: 'Abel'
+          firstName: 'Ashley'
+    meta: [ ]
+
+Delete:
+  'delete one':
+    userId: 1
+    services:
+      performance.performance_tracker_service: \OrangeHRM\Performance\Service\PerformanceTrackerService
+    attributes: [ ]
+    body:
+      ids: [1]
+    query: [ ]
+    data:
+      [1]
+    meta: [ ]
+
+
+
+
+
+
+
+
```

Comment:
Remove blank lines. Keep one new line at the end of file

Reply:
removed